frontend/src/scenes/session-recordings/player/inspector/inspectorListFiltering.test.ts

```diff
@@ -0,0 +1,182 @@
+import { filterInspectorListItems } from 'scenes/session-recordings/player/inspector/inspectorListFiltering'
+import {
+    InspectorListBrowserVisibility,
+    InspectorListItemDoctor,
+    InspectorListItemEvent,
+    InspectorListOfflineStatusChange,
+} from 'scenes/session-recordings/player/inspector/playerInspectorLogic'
+import { SharedListMiniFilter } from 'scenes/session-recordings/player/playerSettingsLogic'
+
+import { PerformanceEvent, SessionRecordingPlayerTab } from '~/types'
+
+describe('filtering inspector list items', () => {
```

Comment:
moved tests out into their own file... should have done that in previous pr really